menas-web/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# To build, first build menas and be in enceladus/menas folder. Then run:
+# $> docker build -t menas .
+# Run image using:
+# $> docker run \
+#       -e JAVA_OPTS=' \
+#           -Dmenas.mongo.connection.string=mongodb://host.docker.internal:27017 \
+#           -Dmenas.mongo.connection.database=menas \
+#       -p 8080:8080 \
+#       menas
```

Comment:
You know when you write something just so you don't forget and then it propagates everywhere. That is the history of this comment. Probably shouldn't be here but in some documentation page.